src/main/core/API/Finder/Log/SecurityLogFinder.php

```diff
@@ -26,17 +26,23 @@ public function configureQueryBuilder(QueryBuilder $qb, array $searches = [], ar
     {
         foreach ($searches as $filterName => $filterValue) {
             switch ($filterName) {
-                case 'workspace':
-                    $qb->leftJoin('obj.workspace', 'w');
-                    $qb->andWhere('w.uuid = :workspaceId');
-                    $qb->setParameter('workspaceId', $filterValue);
+                case 'user':
+                    $qb->leftJoin('obj.doer', 'd');
+                    $qb->andWhere('d.uuid = :id');
+                    $qb->setParameter('id', $filterValue);
+                    break;
+                case 'email':
+                    $qb->orWhere('obj.details LIKE :email');
+                    $qb->setParameter('email', '%'.$filterValue.'%');
                     break;
 
                 default:
                     $this->setDefaults($qb, $filterName, $filterValue);
             }
         }
 
+        $qb->orderBy('obj.id', 'desc');
```

Comment:
You should initialize the default sort in UI : https://github.com/claroline/Claroline/blob/13.x/src/main/core/Resources/modules/administration/community/user/store/reducer.js#L15. Otherwise users will not be able to change the sort of the list.

Reply:
Change done :D 

Thx